code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -251,17 +251,13 @@
 	L.SetStuttering(4 SECONDS)
 
 	ADD_TRAIT(L, TRAIT_WAS_BATONNED, user_UID) // so one person cannot hit the same person with two separate batons
-	L.apply_status_effect(STATUS_EFFECT_DELAYED, 2 SECONDS, CALLBACK(L, TYPE_PROC_REF(/mob/living, KnockDown), knockdown_duration), COMSIG_LIVING_CLEAR_STUNS)
 	addtimer(CALLBACK(src, PROC_REF(baton_delay), L, user_UID), 2 SECONDS)
 
 	SEND_SIGNAL(L, COMSIG_LIVING_MINOR_SHOCK, 33)
 
 	L.lastattacker = user.real_name
 	L.lastattackerckey = user.ckey
-	L.visible_message("<span class='danger'>[user] has stunned [L] with [src]!</span>",
```

Comment:
Any specific reason to remove this message?

Reply:
It felt weird seeing these messages in the chat:
```
X has thrown the stunbaton
X has stunned Y with the stunbaton
Y has been hit with the stunbaton
```
Imo the first and last of these are good enough to indicate what happened, but if you disagree I'm also fine with adding it back in

Reply:
I was thinking something like `the stunbaton shocks Y` or something like that, but honestly with the sound-addition back, its fine.

Reply:
Oh good call, I'll add it back in